package.json

```diff
@@ -41,17 +41,14 @@
     "yargs": "^6.6.0"
   },
   "devDependencies": {
-    "babel-cli": "^6.18.0",
-    "babel-eslint": "^7.1.1",
-    "babel-jest": "^20.0.3",
-    "babel-plugin-transform-object-rest-spread": "6.20.2",
-    "babel-preset-env": "^1.1.11",
-    "eslint": "^3.13.1",
-    "eslint-config-babel": "^6.0.0",
-    "eslint-plugin-flowtype": "^2.30.0",
+    "@types/jest": "^19.2.3",
+    "@types/node": "^7.0.22",
     "fs-extra": "^2.0.0",
     "jest": "^20.0.4",
-    "rimraf": "^2.5.4"
+    "rimraf": "^2.5.4",
+    "ts-jest": "^20.0.4",
+    "tslint": "^5.3.2",
```

Comment:
I was going to say you can try eslint + typescript parser + prettier now instead but can do that later